_posts/en/newsletters/2019-10-30-newsletter.md

```diff
@@ -0,0 +1,152 @@
+---
+title: 'Bitcoin Optech Newsletter #70'
+permalink: /en/newsletters/2019/10/30/
+name: 2019-10-30-newsletter
+slug: 2019-10-30-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the latest C-Lightning release,
+requests help testing a Bitcoin Core release candidate,
+describes discussions about simplified LN commitments using CPFP
+carve-out, and summarizes several top-voted questions and answers from
+the Bitcoin StackExchange.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade to C-Lightning 0.7.3:** this latest [release][c-lightning
+  0.7.3] adds support for a PostgresSQL backend, makes it possible to
+  send funds directly to a particular address when closing a channel,
+  and allows you keep your HD wallet seed encrypted when `lightningd`
+  isn't running---plus many other features and several bug fixes.
+
+- **Help Bitcoin Core release candidate:** experienced users are encouraged to
+  help test the latest release candidates for the upcoming version of
+  [Bitcoin Core][Bitcoin Core 0.19.0].
```

Comment:
Perhaps provide a link to the [testing meta-issue](https://github.com/bitcoin/bitcoin/issues/17079) for test reports (or filing a separate issue in case of problems found).